.gitignore

```diff
@@ -1 +1,6 @@
 .idea
+.vscode
+.env
+.composer
+
+vendor
```

Comment:
It's a good custom to separate directories and files and sort these entries alphabetically.